lib/types/beacon_chain/execution_payload.ex

```diff
@@ -50,4 +55,25 @@ defmodule Types.ExecutionPayload do
   def decode(%__MODULE__{} = map) do
     Map.update!(map, :base_fee_per_gas, &Ssz.decode_u256/1)
   end
+
+  @impl LambdaEthereumConsensus.Container
+  def schema do
+    [
+      {:parent_hash, TypeAliases.hash32()},
+      {:fee_recipient, TypeAliases.execution_address()},
+      {:state_root, TypeAliases.root()},
+      {:receipts_root, TypeAliases.root()},
+      {:logs_bloom, {:bitvector, ChainSpec.get("BYTES_PER_LOGS_BLOOM")}},
+      {:prev_randao, TypeAliases.bytes32()},
+      {:block_number, TypeAliases.uint64()},
+      {:gas_limit, TypeAliases.uint64()},
+      {:gas_used, TypeAliases.uint64()},
+      {:timestamp, TypeAliases.uint64()},
+      {:extra_data, {:bitlist, ChainSpec.get("MAX_EXTRA_DATA_BYTES")}},
+      {:base_fee_per_gas, TypeAliases.uint256()},
+      {:block_hash, TypeAliases.hash32()},
+      {:transactions, {:list, {:bytes, 8}, ChainSpec.get("MAX_TRANSACTIONS_PER_PAYLOAD")}},
```

Comment:
Let's use the transaction type alias here.

Reply:
what would be the `schema` for `Types.Transaction`? I ask because the type alias has to struct fields :thinking:

Reply:
oh I get it now? :sweat_smile:  `TypeAliases.transaction()`

Reply:
would be like this ok?:
```{:list, {:bytes, 8}, ChainSpec.get("MAX_BYTES_PER_TRANSACTION")}```